python/tank/descriptor/bundle_cache_usage/exception.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2017 Shotgun Software Inc.
```

Comment:
name this file errors.py to be compliant with rest of tk-core base and to follow python standards (exception is a reserved word in python and bad things can happen if you name a file `exception.py`! :P )